src/graph/backend/dnnl/kernels/kernel_base.cpp

```diff
@@ -43,6 +43,24 @@ bool kernel_base_t::enabled_constant_cache() const {
     return enabled;
 }
 
+size_t kernel_base_t::encode_constant_cache_key(
+        const std::vector<tensor_t> &inputs, size_t cache_key) const {
+    // Encode the constant memory address into cache key for differentiation
```

Comment:
With this, the original `constant_key_` is not a cache key anymore. I would suggest to rename these variables as well as the function name of `generate_constant_cache_key`.

Reply:
Thanks for the review! Rename to `const_md_hash_` and `generate_constant_md_hash` accordingly, please review again.